sites/all/modules/publizon/publizon.module

```diff
@@ -559,7 +559,8 @@ function publizon_client_library_load($library_id) {
       }
 
       // Save in db cache.
-      cache_set($cid, $libraries[$library_id], 'cache');
+      $expire = variable_get('publizon_client_library_expire', 24 * 60 * 60);
+      cache_set($cid, $libraries[$library_id], 'cache', REQUEST_TIME + $expire);
```

Comment:
Remember that in Drupal 7 you should check your self if content is expire from cache as it is lazy removed.

Ala: `REQUEST_TIME < $cache->expire` before using `$cache->data`.

Reply:
Fixed in https://github.com/eReolen/base/pull/78/commits/2aa54affeccce6ee6a8db9b3401083860a569d82